tests/braille-specs/de-g0-bidi-specs.yaml

```diff
@@ -276,3 +276,23 @@ tests:
 
 # Misc forward tests
   - [" -abc", " ⠠⠤⠁⠃⠉"]
+
+# midnum and endnum tests
+  - ["1(2", "⠼⠁⠠⠶⠼⠃"]
+  - ["1)2", "⠼⠁⠠⠶⠼⠃"]
+  - ["1(", "⠼⠁⠠⠶"]
+  - ["1)", "⠼⠁⠠⠶"]
+  - ["1[2", "⠼⠁⠠⠶⠼⠃"]
+  - ["1]2", "⠼⠁⠠⠶⠼⠃"]
+  - ["1[", "⠼⠁⠠⠶"]
+  - ["1]", "⠼⠁⠠⠶"]
+  - ["1»", "⠼⠁⠠⠴"]
+  - ["1»", "⠼⠁⠠⠴"]
+  - ["1+2", "⠼⠁ ⠈⠖⠼⠃"]
+  - ["1−2", "⠼⠁ ⠈⠤⠼⠃"]
+  - ["1 − 2", "⠼⠁ ⠈⠤⠼⠃"]
+  - ["1±2", "⠼⠁ ⠈⠖⠤⠼⠃"]
+  - ["1 ± 2", "⠼⠁ ⠈⠖⠤⠼⠃"]
+  - ["1=2", "⠼⠁ ⠈⠶⠼⠃"]
+  - ["1≠2", "⠼⠁ ⠈⠔⠶⠼⠃"]
+  - ["1≡2", "⠼⠁ ⠈⠶⠶⠼⠃"]
```

Comment:
Did you add these to the "bidi" tests on purpose?

Reply:
No, historical accident. I moved them to `de-g0.yaml` now.